code/modules/mob/living/carbon/human/species.dm

```diff
@@ -66,6 +66,8 @@
 	var/attack_verb = "punch"	// punch-specific attack verb
 	var/sound/attack_sound = 'sound/weapons/punch1.ogg'
 	var/sound/miss_sound = 'sound/weapons/punchmiss.ogg'
+	
+	var/knockout = /obj/item
```

Comment:
I am going to repeat myself only once before I stop having any faith in you listening.

> Remove this line